src/components/index/why-join.svelte

```diff
@@ -9,60 +9,60 @@
   <div>
     <h2>why should students join acm?</h2>
     <p>
-      <span class="bold">acm<span class="blue">CSUF</span></span>
+      <span class="brand-em">acm<span class="brand-em">CSUF</span></span>

```

Comment:
Why is there 2 nested spans with the same class? I'm assuming the one inside should have a different class?

Reply:
The current code assumes that the entire naming scheme is entirely bold.  This is incorrect.

With the current design guidelines, the only thing that should have font-weight: bold, are the brandings.  This includes, and is exclusive to, the words, "CSUF", "Dev", "Create", "Algo", and "Buds".  Every other character in the title should have font-weight: 600.  This property also applies to headings and subheadings.  This is a very easy fix that I can send a pull request in for later.

Reply:
Thank you for noticing my oversight, @diamondburned. I created an issue (<https://github.com/EthanThatOneKid/acmcsuf.com/issues/62>) on the project board for you to submit your fixes onto, @mikeploythai.